headers/types/dungeon_mode/dungeon.h

```diff
@@ -1815,25 +1826,19 @@ struct dungeon {
     undefined field_0x2ca65;
     undefined field_0x2ca66;
     undefined field_0x2ca67;
-    undefined field_0x2ca68;
-    undefined field_0x2ca69;
-    undefined field_0x2ca6a;
-    undefined field_0x2ca6b;
-    undefined field_0x2ca6c;
-    undefined field_0x2ca6d;
-    undefined field_0x2ca6e;
-    undefined field_0x2ca6f;
-    undefined field_0x2ca70;
-    undefined field_0x2ca71;
-    undefined field_0x2ca72;
-    undefined field_0x2ca73;
-    undefined field_0x2ca74;
-    undefined field_0x2ca75;
-    undefined field_0x2ca76;
-    undefined field_0x2ca77;
-    undefined field_0x2ca78;
-    undefined field_0x2ca79;
-    undefined field_0x2ca7a;
+    struct dungeon_id_8 fainted_id; // 0x2CA68: Copied from dungeon::id, upon fainting.
+    uint8_t fainted_floor;          // 0x2CA69: Copied from dungeon::floor, upon fainting.
+    // 0x2CA6A: Copy of the fainted monster's held item.
+    struct item fainted_monster_held_item;
+    int fainted_monster_exp; // 0x2CA70: Copy of fainted monster's experience.
+    // 0x2CA74: Copy of the fainted monster's ACTUAL max hp (monster::max_hp_stat +
+    // monster::max_hp_boost)
+    int16_t fainted_monster_actual_max_hp;
```

Comment:
Can we just call this `fainted_monster_max_hp`? No qualifier should imply "actual".